### ISSUE TITLE
Randomizing User Agents

### ISSUE THREAD

**Issue by jeffmicklos** [JeffHenry/InstaCrawler#1]
I am wondering if it would be safer to always run the bot with one user agent, rather than a randomized many. As far as Instagram spam detection goes, I imagine it looks shadier to come from a munch of different IPs/browsers rather than constant source, like a normal user would behave.

What do you think?

**Comment by mefynn** [comment]
Could be true. I don't think they care much about the user agent. They only allow you to like 30 images per service(webstagram etc) per hour. So if we could change the ip every 5-10 likes, that would probably help. 



		Fynn Lehnert - Web Developer - OK-Studios 
fynnlehnert@me.com | +49 171 229 55 38 | fynnlehnert.com

**Comment by JeffHenry** [comment]
Honestly, I don't think changing the user agent matters at this point, I would only say it would be worth using if switching between different accounts to make it look more human-like since individuals don't usually change their browsers every couple of likes. If we are using only one account though, its probably better to stick with one user agent. An option could be added to specify a particular user agent with each account so it is consistent, but I don't really have any interest in rewriting the script for the bot since Instagram limits the services so much now. 

- Jeff Henry

**Comment by jeffmicklos** [comment]
I agree @JeffHenry. I am editing it for my own benefit right now, so it comes from 1 IP and an iPhone user agent. Maybe I will submit a PR with the changes.
